.eslintrc.json

```diff
@@ -38,5 +38,26 @@
     "@typescript-eslint/no-unused-vars": "off",
     "@typescript-eslint/no-use-before-define": "off",
     "@typescript-eslint/no-var-requires": "off"
-  }
+  },
+  "overrides": [
+    {
+      "files": [
+        "plugins/plugin-apache-composer/**/*.ts",
+        "plugins/plugin-bash-like/**/*.ts",
+        "plugins/plugin-core-support/**/*.ts",
+        "plugins/plugin-editor/**/*.ts",
+        "plugins/plugin-grid/**/*.ts",
+        "plugins/plugin-manager/**/*.ts",
+        "plugins/plugin-openwhisk-debug/**/*.ts",
+        "plugins/plugin-openwhisk-editor-extensions/**/*.ts",
+        "plugins/plugin-proxy-support/**/*.ts",
+        "plugins/plugin-tutorials/**/*.ts",
+        "plugins/plugin-wrk/**/*.ts",
+        "plugins/plugin-wskflow/**/*.ts"
```

Comment:
good this be plugins/**/*.ts? don't we also want packages/**/*.ts?

Reply:
The point is to add here one package/plugin, to fix the errors, make a PR and start with the next one.

If we add here all of them now we should have to fix all the errors in the same PR. Which would be the same that enabling the rule globally.

Reply:
i thought unused-vars was done? is the implication that it is not?

Reply:
It isn't

Reply:
There are still many unused variables in the k8s, openwhisk & tekton plugins. And also in the packages.